kernel/shared/src/main/scala/cats/effect/kernel/GenTemporal.scala

```diff
@@ -72,6 +72,13 @@ trait GenTemporal[F[_], E] extends GenConcurrent[F, E] with Clock[F] {
     timeoutTo(fa, duration, timeoutException)
   }
 
+  /**
+   * Returns an effect that completes with the result of the source together
+   * with the duration that it took to complete.
+   */
+  def timed[A](fa: F[A]): F[(FiniteDuration, A)] =
+    map3(realTime, fa, realTime)((startTime, a, endTime) => (endTime.minus(startTime), a))
```

Comment:
Close! We *definitely* want to use `monotonic` here. For starters, it's much faster and higher resolution than `realTime`, but additionally it doesn't suffer from re-sync issues. If someone changes the system time between `startTime` and `endTime`, for example, the result here will be inaccurate. `monotonic` doesn't have that problem

```suggestion
    map3(monotonic, fa, monotonic)((startTime, a, endTime) => (endTime.minus(startTime), a))
```

Also given that we're doing this with `Applicative`, I *think* it can just be on `Clock` instead!

Reply:
Done! Thank you for your feedback

Reply:
Appears I broke something, I'll check a bit later

Reply:
You just need to do `scalafmtAll` in `sbt`.